auth-web/src/routes/index.ts

```diff
@@ -116,9 +116,6 @@ router.beforeEach((to, from, next) => {
         } else {
           return next({ path: `/${Pages.ACCOUNT_FREEZE}` })
         }
-      } else if (currentOrganization?.statusCode === AccountStatus.PENDING_AFFIDAVIT_REVIEW) {
```

Comment:
Moved this logic to common components.

Reply:
this might still be needed for with in app navigations...

Reply:
Good catch! added it back.